components-core/src/main/java/com/adyen/checkout/components/status/StatusRepository.kt

```diff
@@ -43,7 +43,10 @@ class DefaultStatusRepository constructor(
 
     private var delay: Long = 0
 
-    private val refreshFlow: MutableSharedFlow<String> = MutableSingleEventSharedFlow()
+    private val refreshFlow: MutableSharedFlow<String> = MutableSharedFlow(
+        replay = 1,
```

Comment:
We do actually, when I remove it the refresh doesn't trigger anymore. Any idea why?

Reply:
I tried using `MutableSharedFlow(extraBufferCapacity = 1)` which made it work, but ` MutableSharedFlow()` doesn't. WDYT?

Reply:
`extraBufferCapacity` is needed, because we use `tryEmit`. I think its' also better to use `extraBufferCapacity`, because with replay I expect that in some cases the refresh can be triggered multiple times (for example when a device is rotated)